ext/dl_test/.gitignore

```diff
@@ -0,0 +1,41 @@
+*.lo
```

Comment:
Do we really need this file? Isn't that already handled by .gitignore in the root?